localstack/services/dynamodbstreams/dynamodbstreams_api.py

```diff
@@ -133,8 +134,10 @@ def stream_name_from_stream_arn(stream_arn):
 
 
 def random_id(stream_arn, kinesis_shard_id):
+    if six.PY2:
+        kinesis_shard_id = kinesis_shard_id.encode('utf-8')
     namespace = uuid.UUID(bytes=hashlib.sha1(stream_arn.encode('utf-8')).digest()[:16])
-    return uuid.uuid5(namespace, kinesis_shard_id.encode('utf-8')).hex
+    return uuid.uuid5(namespace, kinesis_shard_id).hex
```

Comment:
Can we simply use the following (and import `to_bytes` from `commons` at the top of the file`):
```
return uuid.uuid5(namespace, to_bytes(kinesis_shard_id)).hex
```

Reply:
@whummer That's a more elegant way. I have switched to `to_bytes` and it works in my local environment. Thanks!

Reply:
Great, thanks!